### PR TITLE
refactor: Use ajv format instead of getAddress on all actions

### PR DESCRIPTION
**Pull request by ChaituVR**
Changes:
- Check `format` of `address` and `from` using ajv format https://github.com/snapshot-labs/snapshot.js/blob/master/src/utils.ts#L44-L52
- `getAddress` on all actions won't be useful anymore

### CODE REVIEW

**Comment by wa0x6e** [comment]
We're losing the case formatting, does it matter ?

```js
getAddress("0x8ba1f109551bd432803012645ac136ddd64dba72");
// '0x8ba1f109551bD432803012645Ac136ddd64DBA72'
```

File: src/writer/alias.ts [hunk 1]
```diff
@@ -9,7 +9,7 @@ export async function action(message, ipfs, receipt, id): Promise<void> {
   const params = {
     id,
     ipfs,
-    address: getAddress(message.from),
+    address: message.from,
     alias: getAddress(message.alias),
```
Comment (wa0x6e) [inline src/writer/alias.ts:13]: Why leave `getAddress` on this one ?

Comment (wa0x6e) [reply src/writer/alias.ts:13]: By removing this last one, we could also remove `@ethersproject/address` from package dependency

**Comment by ChaituVR** [comment]
> We're losing the case formatting, does it matter ?
> 
> ```js
> getAddress("0x8ba1f109551bd432803012645ac136ddd64dba72");
> // '0x8ba1f109551bD432803012645Ac136ddd64DBA72'
> ```

Yes yes. Address should be always checksummed, `address` format checks if address is checksummed or not
It will reject the request if address is not checksummed, so we don't need to run `getAddress` on all actions

**Comment by wa0x6e** [comment]
> We're losing the case formatting, does it matter ?
> 
> ```js
> getAddress("0x8ba1f109551bd432803012645ac136ddd64dba72");
> // '0x8ba1f109551bD432803012645Ac136ddd64DBA72'
> ```

In case formatting does matter, we could use `getAddress` only on the writer actions itself, before inserting in the DB, and keep ajv for validation only

File: src/writer/alias.ts (hunk 1 above)
Comment (ChaituVR) [reply src/writer/alias.ts:13]: Agree. Hehe will give it a try

**Review by Todmy — APPROVED** [review]
tACK

**Comment by wa0x6e** [comment]
Should this one also wait on https://github.com/snapshot-labs/snapshot-sequencer/pull/192 ?

**Comment by ChaituVR** [comment]
Updated PR

**Review by wa0x6e — APPROVED** [review]
tAck

**Pull request merged**